integration_tests/CMakeLists.txt

```
@@ -1048,6 +1048,7 @@ RUN(NAME complex_11 LABELS gfortran llvmImplicit wasm)
 RUN(NAME complex_12 LABELS gfortran llvm wasm)
 RUN(NAME complex_13 LABELS gfortran llvm wasm)
 RUN(NAME complex_14 LABELS gfortran llvm wasm)
+RUN(NAME complex_15 LABELS gfortran llvm wasm fortran)
```

The CI failed for WASM, so I would remove it:
```suggestion
RUN(NAME complex_15 LABELS gfortran llvm fortran)
```